src/main/java/org/broadinstitute/dsm/jobs/ExternalShipperJob.java

```diff
@@ -32,12 +27,9 @@ public void execute(JobExecutionContext context) {
                 ExternalShipper shipper = (ExternalShipper) Class.forName(DSMServer.getClassName(kitType.getExternalShipper())).newInstance();//GBFRequestUtil
                 ArrayList<KitRequest> kitRequests = shipper.getKitRequestsNotDone(kitType.getInstanceId());
                 shipper.orderStatus(kitRequests);
-                if (kitRequests != null && !kitRequests.isEmpty()) { // only if there are kits which are not yet having kit_label set
-                    logger.info("Working on " + kitRequests.size() + " incomplete external kits");
-                    Instant now = Instant.now();
-                    Instant dynamicStartTime = now.minus(5, ChronoUnit.DAYS);
-                    shipper.orderConfirmation(kitRequests, dynamicStartTime.toEpochMilli(), now.toEpochMilli());
-                }
+                Instant now = Instant.now();
```

Comment:
removed kitRequests from orderConfirmation because it seemed obsolete

Reply:
Nice!  That will speed things up and free a bunch of memory too.

Reply:
Never mind--now I see that we need it for other things anyway...

Reply:
Maybe add `kitRequests.clear()` after L29 so that we can free this memory before the long running `orderConfirmation()` method, since we don't need the object any more.